docs/content/preview/reference/drivers/csharp/yb-npgsql-reference.md

```diff
@@ -0,0 +1,201 @@
+---
+title: C# Drivers
+linkTitle: C# Drivers
+description: C# Drivers for YSQL
+headcontent: C# Drivers for YSQL
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    name: C# Drivers
+    identifier: ref-yb-npgsql-driver
+    parent: drivers
+    weight: 600
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../yb-npgsql-reference/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YugabyteDB Npgsql Smart Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="../postgres-npgsql-reference/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PostgreSQL Npgsql Driver
+    </a>
+  </li>
+
+</ul>
+
+The [Yugabyte Npgsql smart driver](https://github.com/yugabyte/npgsql) is a distributed .NET driver for [YSQL](../../../api/ysql/) built on the [PostgreSQL Npgsql driver](https://github.com/npgsql/npgsql/tree/main/src/Npgsql), with additional [connection load balancing](../../smart-drivers/) features.
+
+## Fundamentals
+
+Learn how to perform common tasks required for C# application development using the NpgsqlYugabyteDB driver.
+
+### Download the driver dependency
+
+If you are using Visual Studio IDE, add the NpgsqlYugabyteDB package to your project as follows:
+
+1. Right-click **Dependencies** and choose **Manage Nuget Packages**
+1. Search for `NpgsqlYugabyteDB` and click **Add Package**. You may need to click the **Include prereleases** checkbox.
+
+To add the NpgsqlYugabyteDB package to your project when not using an IDE, use the following `dotnet` command:
+
+```csharp
+dotnet add package NpgsqlYugabyteDB
+```
+
+or any of the other methods mentioned on the [nuget page](https://www.nuget.org/packages/Npgsql/) for NpgsqlYugabyteDB.
+
+### Connect to YugabyteDB database
```

Comment:
Don't think we need to repeat all this here; see other smart driver pages for how this is arranged